lib/gears/timer.lua

```diff
@@ -38,7 +38,7 @@
 --            awful.spawn.easy_async(
 --                {"sh", "-c", "acpi | sed -n 's/^.*, \([0-9]*\)%/\1/p'"},
 --                function(out)
---                    mybatterybar.battery = out
+--                    mybatterybar.set_battery(out)
```

Comment:
shouldn't both work?

Reply:
Yes. Like any other widget, this has the `__newindex` handler for it.